cli/rename.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// RenameCommand use to implement 'rename' command, it rename a container.
+type RenameCommand struct {
+	baseCommand
+}
+
+// Init initialize rename command.
+func (rc *RenameCommand) Init(c *Cli) {
+	rc.cli = c
+
+	rc.cmd = &cobra.Command{
+		Use:   "rename [container] [newName]",
+		Short: "Rename a exist container with newName",
```

Comment:
Please add Long description and examples as well, like the create command does: https://github.com/alibaba/pouch/blob/master/cli/create.go#L28 and https://github.com/alibaba/pouch/blob/master/cli/create.go#L33 . 

It help a lot to generate cli docs. It is a feature merged just this morning. So maybe you have no idea about this. 😄